wolfcrypt/test/test.c

```diff
@@ -19746,10 +19746,12 @@ static int ecc_test_key_gen(WC_RNG* rng, int keySize)
         ERROR_OUT(-9640, done);
     }
 
+#ifndef NO_FILESYSTEM
```

Comment:
Isn't this buffer in certs_test.h with USE_CERT_BUFFERS_256?  We should use the cert buffers vs not doing the tests.

Reply:
The problem is test.c does not define eccPubKeyDerFile or eccPkcs8KeyDerFile if NO_FILESYSTEM is defined.  However, it looks like SaveDerAndPem still works with NO_FILESYSTEM and just ignores the middle 2 parameters.  I will update this to run SaveDerAndPem without the missing files if NO_FILESYSTEM is defined.